### ISSUE TITLE
Support embeds as a response type

### ISSUE THREAD

**Issue by philpax** [rustformers/llmcord#21]
#20 brought to my attention that embeds could be used as a way of presenting the response; they have full Markdown support, as well as longer message lengths.

I'd like to add this as an option (making sure to increase the message length) so that users can specify the use of embeds if that works better than raw messages for their use case.

**Comment by pabl-o-ce** [comment]
Just a idea maybe is possible to add the user question message separate from the response message to get the full length of `4096` size.

Only if is possible :)

**Comment by philpax** [comment]
What if the question was in the embed's title? That should be a separate length limit 🤔

**Comment by pabl-o-ce** [comment]
yes and whenever the limit pass the 256 characters just cut the question to adjust to the title length
